aea/manager/manager.py

```diff
@@ -230,13 +230,15 @@ def add_error_callback(
         """Add error callback to call on error raised."""
         self._error_callbacks.append(error_callback)
 
-    def start_manager(self, local: bool = True) -> "MultiAgentManager":
+    def start_manager(
+        self, local: bool = False, remote: bool = False
```

Comment:
notice: default for `local` is now `False`, as we decided to denote mixed mode as `local=False` and `remote=False`